numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -4347,12 +4421,15 @@ intern_strings(void)
     npy_ma_str_ndmin = PyUString_InternFromString("ndmin");
     npy_ma_str_axis1 = PyUString_InternFromString("axis1");
     npy_ma_str_axis2 = PyUString_InternFromString("axis2");
+    npy_ma_str_like = PyUString_InternFromString("like");
+    npy_ma_str_numpy = PyUString_InternFromString("numpy");
 
     return npy_ma_str_array && npy_ma_str_array_prepare &&
            npy_ma_str_array_wrap && npy_ma_str_array_finalize &&
            npy_ma_str_ufunc && npy_ma_str_implementation &&
            npy_ma_str_order && npy_ma_str_copy && npy_ma_str_dtype &&
-           npy_ma_str_ndmin && npy_ma_str_axis1 && npy_ma_str_axis2;
+           npy_ma_str_ndmin && npy_ma_str_axis1 && npy_ma_str_axis2 &&
+           npy_ma_str_like && npy_ma_str_numpy;
```

Comment:
for reviewers: I have an open PR, #17014, which will conflict with this lines

Reply:
I merged that PR already, and it doesn't conflict, because we have this pattern twice, once for ufuncs once for multiarray (historic reasons really), and I think you only changed the umath one.

Reply:
Whoops, totally missed that - thanks!